index.js

```diff
@@ -231,35 +229,27 @@ plugin.uploadImage = function (data, callback) {
 		}
 
 		fs.readFile(image.path, function (err, buffer) {
-			uploadToS3(image.name, err, buffer, callback);
+			var maxWidth = parseInt(meta.config.maximumImageWidth);
```

Comment:
I don't think this is correct. I looks like it will try to resize files? that are not images.